apps/health_campaign_field_worker_app/lib/pages/language_selection.dart

```diff
@@ -30,8 +31,14 @@ class LanguageSelectionPage extends StatelessWidget {
                 if (state is! AppInitialized) return const Offstage();
                 final appConfig = state.appConfiguration;
                 final languages = state.appConfiguration.languages;
-                final localizationModulesList =
-                    state.appConfiguration.backendInterface?.interfaces;
+                final dashboardInterface = Interfaces();
```

Comment:
Create a method and move dashboard related calls

Reply:
This will be removed after adding the dashboard service to service registry in MDMS